### ISSUE TITLE
Not working anymore on Sequoia

### ISSUE THREAD

**Issue by BakaPhoenix** [Marginal/QLVideo#129]
Hello,

Just upgraded to 15.0.1 and the app is no longer working.
Did the basic trouble shooting steps but i see not changes at all.

**Comment by igRo** [comment]
It does generate a thumbnail as a file icon, but there are no longer snapshots displayed when the same file is opened in the Quick Look mode.

![Screenshot 2024-10-23 at 11 46 09](https://github.com/user-attachments/assets/6f7deafb-46ba-4379-ba4c-e3c1392b4f30)
![Screenshot 2024-10-23 at 11 46 23](https://github.com/user-attachments/assets/ce02713d-c483-4599-8793-1140fa8b1978)

**Comment by BakaPhoenix** [comment]
After letting it run for a while and enabling the app in Login Item & Extension
It works

![immagine](https://github.com/user-attachments/assets/65596ea6-138d-4d44-bf31-43c180df7148)

**Comment by abdoufma** [comment]
@BakaPhoenix Just be clear, By "letting it run" do you mean simply launching the `QuickLook Video` app and waiting, running the "refresh" command, or using some CLI command ?

I'm also on Sequoia and only have thumbnails but no previews...

**Comment by tocotronaut** [comment]
maybe you can try to Update ffprobe manually...
ffprobe is to determine the type of video file. 

If you right-click the .app in Finder and use "Show Package" you should see this:
![Bildschirmfoto 2024-11-14 um 21 29 25](https://github.com/user-attachments/assets/717b7bb3-0189-4442-83a0-ceeb463047db)

you can look here: https://ffmpeg.org/download.html
or here: https://evermeet.cx/ffmpeg/
vor new ffprobe download.

Just download ffprobe and overwrite the file inside the app.
(double click it once to see if it runs)

**Comment by abdoufma** [comment]
> > @BakaPhoenix By "letting it run" do you mean simply launching the `QuickLook Video` app and waiting, running the "refresh" command, or using some CLI command ?
> > Sorry I'm not used to this.
> 
> Yes it fixed itself after let it running for some time and enabling the add on in the settings

Yes, but did you just open it, or did you have to click refresh letting it run?

**Comment by gth001** [comment]
I too am having no luck with arm mac sequoia . I've waited a while, enabled all the extension toggles, force quit restarted Finder…

<img width="905" alt="image" src="https://github.com/user-attachments/assets/84382c56-5940-4111-b61b-c6ce61e818d1" />

**Comment by Marginal** [comment]
[Release 2.10](https://github.com/Marginal/QLVideo/releases/latest) improves support on Sequoia:
- Thumbnails for all types of "native" and "non-native" video files should work fine - but you may need to press "Refresh" in the app.
- Previews for `.mkv`, `.webm` and older obscure file types are displayed as cover art or a single snapshot. But previews of the  following "native" filetypes are still handled by macOS: `.avi`, `.dv`, `.m4v`, `.mov`, `.mp2`, `.mp4`, `.mts`, `.ts`, `.vob`. So if macOS can't handle them you will either see a thumbnail or nothing.

Please report how you get on.

**Comment by gth001** [comment]
Thanks - 2.10 fixed my issues :)

**Comment by abdoufma** [comment]
@Marginal This is working better, thanks!

> Previews for .mkv, .webm and older obscure file types are displayed as cover art or a single snapshot

Is it not possible to have a bunch of screenshots you can go through like before on Sequoia?

**Comment by Marginal** [comment]
@abdoufma 
> Is it not possible to have a bunch of screenshots as a preview for an `.mkv` file like before on Sequoia?

Yes it is. I'll restore this functionality in the next release.

**Comment by Marginal** [comment]
Closing this as fixed. Please open a new issue for any ongoing problem.
